tests/python/test_grammar_matcher.py

```diff
@@ -106,13 +96,13 @@ def test_fill_next_token_bitmask(
         assert rejected_sizes[-1] == expected_rejected_sizes[-1]
 
 
-def test_token_operations() -> None:
+def test_token_operations():
     """Test accepting token and finding the next token mask."""
-    # fmt: off
     vocab = [
+        # fmt: off
```

Comment:
I changed these to be external because Ruff ignores them inside the statement. https://docs.astral.sh/ruff/formatter/#format-suppression

Reply:
Thanks for bringing that up. But since we use black as the formatter instead of ruff, and black supports `fmt: off` by lines, this should be fine as well.